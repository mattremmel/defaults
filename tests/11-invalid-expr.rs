use defaults::Defaults;

#[derive(Defaults)]
struct Foo {
    #[default(value = "1/")]
    x: usize,
}

fn main() {}
