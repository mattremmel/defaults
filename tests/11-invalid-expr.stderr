error: Not a valid expression
 --> $DIR/11-invalid-expr.rs:5:23
  |
5 |     #[default(value = "1/")]
  |                       ^^^^
